components/runtime/squeak5c/CDDL.license

```diff
@@ -0,0 +1,377 @@
+COMMON DEVELOPMENT AND DISTRIBUTION LICENSE Version 1.0
```

Comment:
Where is it used?

Reply:
The Makefile(s) of the squeak(-4,-5,-5c) packages are contributed and use that license.

Reply:
There's no need to add it explicitly here. There's reference to it in Makefile and manifests.

Reply:
It shouldn't appear in component's directory.